app/controllers/coronavirus_form/additional_product_controller.rb

```diff
@@ -1,36 +1,28 @@
 # frozen_string_literal: true
 
 class CoronavirusForm::AdditionalProductController < ApplicationController
-  before_action :check_first_question_answered, only: :show
-
-  def show
-    render "coronavirus_form/#{PAGE}"
-  end
-
   def submit
     additional_product = strip_tags(params[:additional_product]).presence
 
     invalid_fields = validate_radio_field(
-      PAGE,
+      controller_name,
       radio: additional_product,
     )
 
     if invalid_fields.any?
       flash.now[:validation] = invalid_fields
-      render "coronavirus_form/#{PAGE}"
+      render controller_path
     elsif additional_product == I18n.t("coronavirus_form.questions.additional_product.options.option_yes.label")
-      redirect_to controller: "coronavirus_form/medical_equipment_type", action: "show"
+      redirect_to medical_equipment_type_path
     elsif session["check_answers_seen"]
-      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+      redirect_to check_your_answers_path
     else
-      redirect_to controller: "coronavirus_form/hotel_rooms", action: "show"
+      redirect_to hotel_rooms_path
     end
   end
 
 private
 
-  PAGE = "additional_product"
-
   def previous_path
```

Comment:
Can the `path` helper be used in this method too? 
e.g. 
`product_details_path(product_id: latest_product_id)`
rather than:
`"/product-details?product_id=#{latest_product_id}"`

Reply:
Thanks, I missed that.